numpy/lib/stride_tricks.py

```diff
@@ -197,6 +197,43 @@ def _broadcast_shape(*args):
     return b.shape
 
 
+@set_module('numpy')
+def broadcast_shapes(*args):
+    """
+    Get Broadcast shape from the given shape tuples.
+
+    Parameters
+    ----------
+    `*args` : tuples of ints, or ints
+        The shapes to be broadcast against each other.
+
+    Returns
+    -------
+    tuple
+        Broadcasted shape.
+
+    Raises
+    ------
+    ValueError
+        If the shapes are not compatible and cannot be broadcast according
+        to NumPy's broadcasting rules.
+
+    Notes
+    -----
+    .. versionadded:: 1.20.0
```

Comment:
I think we normally put `versionadded` at the top of the docstring before the parameters.

Reply:
OK great, that is what @charris also indicated above, but I then copied the convention from `broadcast_to` in the same file and some other functions I spot checked. I will fix this. 
I read the [guidelines here](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard), but `versionadded` is not included.